types/index.d.ts

```diff
@@ -1,5 +1,5 @@
 export interface ColorPalette {
-	[key: string]: string | ColorRange
+	[key: string]: any
```

Comment:
Does this type erasure do anything? Tailwind only allows colors to be defined as a string or single layer object?

Reply:
Figured out why when upgrading dependencies. I fixed the type errors in tests with 11e0f5fd93a006a5107e722b69bfc5f311203bf7.